clinics/manager/manager.go

```diff
@@ -219,6 +224,32 @@ func (c *manager) getDemoPatient(ctx context.Context) (*patients.Patient, error)
 	return patient, nil
 }
 
+func (c *manager) deleteClinic(ctx context.Context, clinicId string) error {
+	filter := patients.Filter{ClinicId: &clinicId}
+	pagination := store.Pagination{Limit: 2}
+	res, err := c.patientsService.List(ctx, &filter, pagination, nil)
+
+	if err != nil {
+		return err
+	}
+	if res == nil {
+		return fmt.Errorf("patient list result not defined")
```

Comment:
Is there a reason sometimes the error is defined in the `errors` package and sometimes defined inline?

Reply:
The errors in the `errors` package ought to wrapped so when they bubble up to the API layer we return the correct http code. All other errors which bubble up with result in a 500 response, which is intended here.